### DISCUSSION TITLE
I managed to use FIEF in Metabase. It's one of the best open-source BI system

### DISCUSSION THREAD

**Discussion by rzykov** [fief-dev/fief#473] [Show and tell]
It solved my pain with Metabase! The complete instruction is published there:
https://github.com/rzykov/metabase/blob/fiev_auth/Fief_auth.md

**Comment by fief-bailiff** [comment]
Hail, @rzykov 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to [star our repository](https://github.com/fief-dev/fief) ⭐️

<p align="center">
<img width="656" alt="star-fief" src="https://github.com/fief-dev/fief/assets/1144727/b2db7f59-102f-47e5-9751-67fa10372c52">
</p>

## Want to support us?

Subscribe to one of our paid plan to help us continue our work and receive exclusive information and benefits! Starts at $5/month 🪙

<p align="center">
    <a href="https://polar.sh/fief-dev">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/embed/subscribe.svg?org=fief-dev&darkmode=1">
      <img alt="Subscribe" src="https://polar.sh/embed/subscribe.svg?org=fief-dev">
    </picture>
    </a>
</p>

Farewell!
